book/guided-tour.md

```diff
@@ -154,13 +203,24 @@ slightly awkward `/.` operator. In the following example, we open the
 are designed to be used in this kind of context. Note that this causes the
 standard int-only arithmetic operators to be shadowed locally.
 
-<link rel="import" href="code/guided-tour/main.mlt" part="6.1" />
+```ocaml
+# ;;
```

Comment:
This looks messed up --- the ordering of the lines is backward

Reply:
Thanks, fixed